Makefile

```diff
@@ -227,9 +249,33 @@ lint-monitoring:
 clean:
 	rm -rf $(OUTPUT_DIR)
 
+# Target to create a new Docker Buildx builder
+create-builder:
+	./hack/build-multiarch-operator.sh ${DOCKER_BUILDER}
+
+# Target to build the operator image with support for multiple platforms
+build-multiarch-operator-image:
+	# Remove any existing manifest and image
```

Comment:
please drop the comments (all over unless it is not trivial)

Reply:
Done.

Reply:
please remove more comments, there are still some trivial ones
we usually add as few as possible
only for non trivial stuff